js/bybit.js

```diff
@@ -2025,14 +2017,13 @@ module.exports = class bybit extends Exchange {
         };
         let market = undefined;
         if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchMyTrades() requires a symbol argument');
```

Comment:
You have tested and it is really required, right? If that is the case, we should move this check to the top of this method, cause if the `symbol` argument is missing but required, we should throw immediately.